tests/event-api-tests/search-events/after.spec.ts

```
@@ -0,0 +1,174 @@
+}
+
+test("Check filtering events by 'after' param", async ({ request }) => {
+  const feed = "pdc";
+  // Get all events (e.g., limit 1000)
+  const allEventsResp = await fetchEventsWithParams({
+    params: { feed, limit: 1000 },
+    request,
+  });
+  checkSuccessfulResponse(allEventsResp);
+
+  const allEvents = allEventsResp.json!.data;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)